src/services/item/plugins/importExport/service.ts

```diff
@@ -300,37 +288,31 @@ export class ImportExportService {
   async export(
     actor: Actor,
     repositories: Repositories,
-    { itemId, reply }: { itemId: UUID; reply: FastifyReply },
+    {
+      item,
+      reply,
+      fileStorage,
+    }: { fileStorage: string; item: Item; reply: FastifyReply; log?: FastifyBaseLogger },
   ) {
-    // check item and permission
-    const item = await this.itemService.get(actor, repositories, itemId);
-
```

Comment:
Is the check not required anymore or was it moved?

Reply:
It moved to the controller (I needed the name, it's also better to check the permission before creating the filestorage).